storm-server/src/main/java/org/apache/storm/daemon/supervisor/Slot.java

```diff
@@ -936,6 +936,9 @@ static DynamicState handleRunning(DynamicState dynamicState, StaticState staticS
             }
             dynamicState = dynamicState.withProfileActions(mod, modPending);
         }
+
+        dynamicState.container.processMetrics();
```

Comment:
This is something I just noticed.  `handleRunning` should be called about once a second.  I don't think we want to send metrics that frequently.  Most of the metrics are on a 30 second to 1 min frequency.  I think standardizing on 1 min for us is going to make storing and querying them a lot simpler.  Could we have a small amount of state that we can use to throttle this to once a min?

Reply:
Yes, will fix.